src/SignalR/clients/ts/FunctionalTests/yarn.lock

```diff
@@ -1258,15 +1256,10 @@ minimatch@^3.0.2, minimatch@^3.0.4:
   dependencies:
     brace-expansion "^1.1.7"
 
-minimist@^1.2.5:
-  version "1.2.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
-  integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
-
-minimist@~0.0.1:
-  version "0.0.10"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.0.10.tgz#de3f98543dbf96082be48ad1a0c7cda836301dcf"
-  integrity sha1-3j+YVD2/lggr5IrRoMfNqDYwHc8=
+minimist@>=1.2.6, minimist@^1.2.5, minimist@~0.0.1:
+  version "1.2.6"
```

Comment:
Safe❔

Reply:
Are you asking about the bump from 0.0.1? 1.2.6 is the minimum version in any band that satisfies CG, so we don't have much of a choice. @BrennanConroy are you concerned about this update?

Reply:
It is rare to see these small dependency updates break anything and it's especially not a concern for test code. Since if it breaks we'll know about it 😆